challenges-webapp/src/main/webapp/vue-app/challenges/components/AnnouncementDrawer.vue

```diff
@@ -42,13 +42,17 @@
         </v-chip>
       </div>
       <div class="pl-4 pr-4 pt-9 py-4 my-2">
-        <challenge-description
-          ref="challengeDescription"
+        <div class="py-1 px-2 subtitle-1">
+          * {{ $t('challenges.label.describeYourAchievement') }}
```

Comment:
```suggestion
          {{ $t('challenges.label.describeYourAchievement') }} *
```
The mandatory tip goes after the field name.